### PR TITLE
Added a test section

### PR DESCRIPTION
**Pull request by kdhamric**
Added a test section

This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ Give us a star on Github if you're interested in the project!
 
 Is available at [https://kubeshop.github.io/tracetest](https://kubeshop.github.io/tracetest)
 
+# Tests
+
+We strive to produce quality code and to be able to improve Tracetest rapidly, and safely. Thus, we have a full suite of both frontend and backend code. We are using Cypress for our frontend code, and, surprise surprise, Tracetest for our backend code. You can see the [test runs here](https://github.com/kubeshop/tracetest/actions/workflows/pull-request.yaml), and a blog post describing our [testing pipelines here](https://kubeshop.io/blog/integrating-tracetest-with-github-actions-in-a-ci-pipeline).
```
Comment (jfermi) [inline README.md:119]:
```suggestion
We strive to produce quality code and improve Tracetest rapidly and safely. Therefore, we have a full suite of both frontend and backend code. We are using Cypress for our frontend code and (surprise surprise) Tracetest for our backend code. You can see the [test runs here](https://github.com/kubeshop/tracetest/actions/workflows/pull-request.yaml), and a blog post describing our [testing pipelines here](https://kubeshop.io/blog/integrating-tracetest-with-github-actions-in-a-ci-pipeline).
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by mathnogueira — APPROVED** [review]
LGTM

**Review by schoren — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -114,6 +114,10 @@ Give us a star on Github if you're interested in the project!
 
 Is available at [https://kubeshop.github.io/tracetest](https://kubeshop.github.io/tracetest)
 
+# Tests
+
+We strive to produce quality code and improve Tracetest rapidly and safely. Therefore, we have a full suite of both frontend and backend tests. We are using Cypress to test our frontend code and (surprise surprise) Tracetest for our backend code. You can see the [test runs here](https://github.com/kubeshop/tracetest/actions/workflows/pull-request.yaml), and a blog post describing our [testing pipelines here](https://kubeshop.io/blog/integrating-tracetest-with-github-actions-in-a-ci-pipeline).
```
Comment (jfermi) [inline README.md:119]:
```suggestion
We strive to produce quality code and improve Tracetest rapidly and safely. Therefore, we have a full suite of both frontend and backend tests. We are using Cypress to test our frontend code and (surprise, surprise) Tracetest for our backend code. You can see the [test runs here](https://github.com/kubeshop/tracetest/actions/workflows/pull-request.yaml), and a blog post describing our [testing pipelines here](https://kubeshop.io/blog/integrating-tracetest-with-github-actions-in-a-ci-pipeline).
```

**Review by jfermi — APPROVED** [review]

**Pull request merged**